tests/test_interrupted_sync.py

```diff
@@ -62,26 +62,45 @@ def get_bookmark_value(self, state, stream):
         return None
 
     ##########################################################################
-    # Tests To Override
+    # Tests and Functions To Override
```

Comment:
There's no longer any tests being overridden

Reply:
I guess this can be a section header if we do it again in the future

Reply:
Yea, I left the section header just in case